### ISSUE TITLE
Glass panes and horizontal glass panes.

### ISSUE THREAD

**Issue by lsi5974** [Lykrast/Glassential#1]
If you get around to adding panes to the mod, it would be a little more popular if they could also be placed into a crafting table and made into horizontal glass panes.  This mod does it wonderfully, just not with any special glass.

https://www.curseforge.com/minecraft/mc-mods/horizontal-glass-panes

**Comment by Zygus42** [comment]
Yep, would love to see panes added as a feature also.  Don't really care about the horizontal part as I've always had issues with that mod and no longer use it.

**Comment by Yourname942** [comment]
Please  add this feature to 1.12.2!
